js/chatbot/src/ChatBot.svelte

```diff
@@ -1,8 +1,16 @@
 <script lang="ts">
+	import { marked } from "marked";
+	import Prism from "prismjs";
+	import "prismjs/components/prism-python";
```

Comment:
@pngwn I switched to `prism`, but I can't find a way to easily load additional languages without having to import each individually. Seems like `loadlanguages()` doesn't work.